tt-train/tests/model/gpt2s_test.cpp

```diff
@@ -0,0 +1,86 @@
+// SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include <gtest/gtest.h>
+
+#include <core/ttnn_all_includes.hpp>
+
+#include "autograd/auto_context.hpp"
+#include "core/compute_kernel_config.hpp"
+#include "core/tt_tensor_utils.hpp"
+
+enum class ExpectedResult { OK, ERROR };
+
+struct MatmulInput {
+    ttnn::Shape shape_a;
+    ttnn::Shape shape_b;
+    bool transpose_a{false};
+    bool transpose_b{false};
+};
+
+struct MatmulTest {
+    MatmulInput input;
+    ExpectedResult expected_result;
+};
+
+TEST(GPT2STest, Matmul) {
```

Comment:
add comment that it is a gpt2 test with bs64

Reply:
done